ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -187,6 +187,45 @@ device_classes: !ZenPackSpec
                         datapoints:
                             status:
                                 rrdmin: 0
+                    iostats:
```

Comment:
I'd recommend renaming this datasource and the parser to diskstats so users could more easily intuit what it is.